<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminwindow</class>
 <widget class="QMainWindow" name="adminwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1147</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="boton_atras">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#E6FAFC;
font: 87 16pt &quot;Segoe UI Black&quot;;

background-color: rgb(85, 0, 127);</string>
    </property>
    <property name="text">
     <string>atras</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>731</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="90,25">
     <item>
      <widget class="QFrame" name="infobox_3">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-width: 0px;
border-style:solid;
</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="username_label_3">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);
font: 87 35pt &quot;Segoe UI Black&quot;;</string>
          </property>
          <property name="text">
           <string>Usuario</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="name_label_3">
          <property name="styleSheet">
           <string notr="true">font: 87 20pt &quot;Segoe UI Black&quot;;
color: rgb(91, 42, 134);</string>
          </property>
          <property name="text">
           <string>Nombre</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="icon_label_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>370</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_config">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#E6FAFC;
font: 87 16pt &quot;Segoe UI Black&quot;;

background-color: rgb(85, 0, 127);</string>
    </property>
    <property name="text">
     <string>usuarios</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1147</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
